modules/SecondaryPool/src/lib.rs

```diff
@@ -30,37 +35,93 @@ fn map_pools(blk: eth::Block) -> Result<Pools, substreams::errors::Error> {
     })
 }
 
+#[substreams::handlers::store]
+pub fn store_pools_created(pools: Pools, store: StoreSetProto<Pool>) {
+    for pool in pools.pools {
+        let pool_address = &pool.pool_address;
+        // store.set(1, format!("pool:"), &pool);
+        store.set(1, &String::from_utf8_lossy(&pool_address), &pool);
+    }
+}
+
 #[substreams::handlers::map]
-fn map_trades(
+fn map_subscriptions(
```

Comment:
@yaksheshgupta 

does map_subscriptions exist in .yaml ? I see only map_trades. Besides, secondary pools have trades, not subscriptions